pkgs/development/python-modules/array-api-strict/default.nix

```
@@ -39,11 +39,11 @@ buildPythonPackage rec {
 
   passthru.updateScript = nix-update-script { };
 
-  meta = with lib; {
+  meta = {
     homepage = "https://data-apis.org/array-api-strict";
     changelog = "https://github.com/data-apis/array-api-strict/releases/tag/${version}";
     description = "A strict, minimal implementation of the Python array API";
-    license = licenses.bsd3;
-    maintainers = [ maintainers.berquist ];
+    license = lib.licenses.bsd3;
+    maintainers = [ lib.maintainers.berquist ];
```

```suggestion
    maintainers = with lib.maintainers; [ berquist ];
```

Optional style change, not a blocker. See the discussion in #331085 and choose whether you want to keep this as-is, or use `with`.